package/upgrade/migrations/managed_charts/v1.0.0.sh

```diff
@@ -16,9 +16,18 @@ add_longhorn_settings()
   yq e '.spec.values.longhorn.defaultSettings.defaultDataPath = "/var/lib/harvester/defaultdisk"' $CHART_MANIFEST -i
 }
 
+adjust_node_exporter_limits()
+{
+  # Increase limits.memory (https://github.com/harvester/harvester-installer/pull/250)
+  yq e '.spec.values.prometheus-node-exporter.resources = {"limits": {"cpu": "200m", "memory": "180Mi"}, "requests": {"cpu": "100m", "memory": "30Mi"}}' $CHART_MANIFEST -i
+}
+
```

Comment:
The Prometheus resource config is also changed per https://github.com/harvester/harvester-installer/pull/240/files, also just wondering if we can build a reusable mechanism through the upgrade controller instead of dealing it case by case, e.g compare the latest manifest YAML with the applied managedChart and patch the values accordingly?

Reply:
These settings are recorded in rancherd bootstrap resources templates and applied by rancherd during bootstrap.
It's not visible to the backend controller.
A better way to prevent this kind of hotfix is to adjust those settings within charts' default values files. Since these PRs adjust the override values in the ManagedChart, we can only patch the ManagedChart spec.values to be consistent.

Reply:
If we want to go in this way, we might want to reuse Helm's 3-way merge or `kubectl patch` merge strategy. And that makes me feel the responsibility should be on managedchart side? Not sure about that.

Reply:
@guangbochen Add [Prometheus pod change](https://github.com/harvester/harvester/compare/950dde99b8ae63a5e07330fc7883c53199d8bb9e..80eed1b3549d9f0744b1e03235bb4671dec125e7) as well.